From 679e5361211dfde27fd0119d0f56cef3dc5f383d Mon Sep 17 00:00:00 2001
From: Jan Grulich <jgrulich@redhat.com>
Date: Thu, 27 Jul 2023 12:35:44 +0200
Subject: [PATCH 06/19] GTK3 theme: simplify code

There's no need to first convert to QString and then convert back to
QByteArray.
---
 src/plugins/platformthemes/gtk3/qgtk3theme.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/platformthemes/gtk3/qgtk3theme.cpp b/src/plugins/platformthemes/gtk3/qgtk3theme.cpp
index 2eaba23beb..39cdc40390 100644
--- a/src/plugins/platformthemes/gtk3/qgtk3theme.cpp
+++ b/src/plugins/platformthemes/gtk3/qgtk3theme.cpp
@@ -135,7 +135,7 @@ QGtk3Theme::QGtk3Theme()
         if (qEnvironmentVariableIsEmpty("XCURSOR_SIZE")) {
             const int cursorSize = gtkSetting<gint>("gtk-cursor-theme-size");
             if (cursorSize > 0)
-                qputenv("XCURSOR_SIZE", QString::number(cursorSize).toUtf8());
+                qputenv("XCURSOR_SIZE", QByteArray::number(cursorSize));
         }
         if (qEnvironmentVariableIsEmpty("XCURSOR_THEME")) {
             const QString cursorTheme = gtkSetting("gtk-cursor-theme-name");
-- 
2.43.0

